axelrod/tests/test_tournament.py

```diff
@@ -29,28 +57,244 @@ def test_init(self):
         anonymous_tournament = axelrod.Tournament(players=self.players)
         self.assertEqual(anonymous_tournament.name, 'axelrod')
 
-    def test_tournament(self):
-        """Test tournament."""
+    def test_serial_play(self):
+        tournament = axelrod.Tournament(
+            name=self.test_name,
+            players=self.players,
+            game=self.game,
+            turns=200,
+            repetitions=self.test_repetitions)
+        scores = tournament.play().scores
+        actual_outcome = sorted(zip(self.player_names, scores))
+        self.assertEqual(actual_outcome, self.expected_outcome)
+
+    def test_parallel_play(self):
+        tournament = axelrod.Tournament(
+            name=self.test_name,
+            players=self.players,
+            game=self.game,
+            turns=200,
+            repetitions=self.test_repetitions,
+            processes=2)
+        scores = tournament.play().scores
+        actual_outcome = sorted(zip(self.player_names, scores))
+        self.assertEqual(actual_outcome, self.expected_outcome)
```

Comment:
Actually - it's doing more than checking that it doesn't crash. It's checking that we get the expected result too - but that could have been produced by any number of processes.